frontend/util.lua

```diff
@@ -81,12 +81,16 @@ function util.secondsToClock(seconds, withoutSeconds)
         end
     else
         local round = withoutSeconds and require("optmath").round or math.floor
-        local hours = string.format("%02.f", math.floor(seconds / 3600));
-        local mins = string.format("%02.f", round(seconds / 60 - (hours * 60)));
+        local hours = string.format("%02.f", math.floor(seconds / 3600))
```

Comment:
I was staring at this for seconds before I realized the change is that you removed the `;` that came from the original gist. There are a few more lines above with the same "defect" compared to our code style. ;-)